code/game/objects/effects/effect_system/effects_foam.dm

```diff
@@ -54,18 +54,17 @@
 		kill_foam()
 		return
 
-	var/fraction = 1/initial(lifetime)
```

Comment:
with current foam settings, a 50u foam reaction with 100u of CLF3 burned a single tile. the fraction stuff is too strong, especially considering that lifetime was cranked up for fastprocess foam to work. since giving up fastspread foam would be an even bigger nerf, the solution is to increase the reagents spread.

ps. why is it divided by lifetime anyways, fuck